src/index.html

```diff
@@ -4,19 +4,109 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="use-credentials"
```

Comment:
The `crossorigin` attribute should be set to `anonymous` instead of `use-credentials` when preconnecting to Google Fonts. This is because Google Fonts does not require credentials, and using `anonymous` is more secure and compatible.